session_schema_test.go

```diff
@@ -188,7 +189,8 @@ func TestCustomTableName(t *testing.T) {
 func TestDump(t *testing.T) {
 	assert.NoError(t, prepareEngine())
 
-	fp := testEngine.Dialect().URI().DbName + ".sql"
+	whole := testEngine.Dialect().URI().DbName
+	fp := whole[:strings.Index(whole, "?")] + ".sql"
```

Comment:
The `DbName` has been parsed. The index ? is unnecessary.

Reply:
But it doesn't work for me (Windows OS, go 11). Here's the error when changes not performed.
> open ./test.db?cache=shared&mode=rwc: The filename, directory name, or volume label syntax is incorrect.
The file name is quite wired here. I think this might be a compatibility problem.

Reply:
Solved in other PR